rep-2000.rst

```diff
@@ -547,6 +547,124 @@ Build System Support:
 - cmake
 - setuptools
 
+Foxy Fitzroy (May 2020 - May 2023)
+----------------------------------
+
+Targeted platforms:
+
++--------------+-----------------------+----------------------+----------------------+--------------------+---------------+
+| Architecture | Ubuntu Focal (20.04)  | MacOS Mojave (10.14) | Windows 10 (VS2019)  | Debian Buster (10) | OpenEmbedded  |
```

Comment:
Catalina? Maybe with a * marking it as tentative?

Reply:
Done 1f61d11